doc/manual.rst

```diff
@@ -4803,7 +4803,6 @@ type class           matches
 ``set``              any set type
 ``seq``              any seq type
 ``auto``             any type
-``any``              distinct auto (see below)
```

Comment:
ok, that's also the only reference to `any` I found in in manual.rst

Reply:
Hard to find, huge file, theres like 90+ "any" but all of them are not from the Type Any but from any as in English word any.
:(

Reply:
Those should be removed when removing `any`, for now is Ok, because is Deprecation only.